gdx/src/com/badlogic/gdx/graphics/g3d/loader/ObjLoader.java

```diff
@@ -439,4 +420,58 @@ public ModelMaterial getMaterial (final String name) {
 		materials.add(mat);
 		return mat;
 	}
-}
+
+	private static class CurrentMaterial {
```

Comment:
ObjMaterial would be a better name imo.

also, it would be better to avoid code duplication by calling reset in contructor.